### ISSUE TITLE
大佬你好，我尝试使用你的代码微调Qwen2.5-7B-Instruct，但是训完后参数没有更新

### ISSUE THREAD

**Issue by PennyYu123** [NovaSearch-Team/RAG-Retrieval#74]
我运行的是train_embedding.sh，检查了原始Qwen2.5-7B-Instruct模型和保存后模型在encode同一个句子的输出，结果几乎一样；另外我在训练时打印GPU0上的某个固定位置的参数，并没有随着optimizer.step()发生更新，问下是我哪里配置的不对吗？

**Comment by PennyYu123** [comment]
解决了，训练llm时一定要改成bf16，不然底层容易溢出，loss scale特别大造成参数基本不更新了
